src/styles/sidebar-styles.js

```diff
@@ -57,8 +57,8 @@ $_documentContainer.innerHTML = `<dom-module id="sidebar-styles">
             }
 
             .panel-menus > iron-selector a.iron-selected {
-                border-left: 6px solid rgb(27, 67, 133);
-                color: rgb(27, 67, 133);
+                border-left: 6px solid #230871;
+                color: #230871;
```

Comment:
<img width="247" alt="Zrzut ekranu 2024-05-25 o 23 09 48" src="https://github.com/FACT-Finder-Web-Components/documentation/assets/37622914/c6833d6b-9afb-4142-9f9b-3a6b8ac6b7c6">

these panels weren't mentioned in the design presentation, but I think they look better with the new color, too. Let me know if it should stay like that.

Reply:
Looks good! Let's keep it.